### ISSUE TITLE
Better mobile support 

### ISSUE THREAD

**Issue by martinklepsch** [octobox/octobox#542]
Are you experiencing this issue in octobox.io or your own instance?
 - [x] octobox.io

🐞 **Problem**
Using Octobox on mobile is already quite nice but could be improved. After adding the app to the homescreen it currently still opens as a tab in Safari and the address bar is always shown taking away screen estate.

🎯 **Goal**

Support various "Progressive Web App" features such as having a "separate application" for octobox when switching between apps and don't show address bar since we don't really need it.

https://nomadlist.com/ is an example which implements these features (checked on ios, not sure about Android) and could be used for comparison.

💡 **Possible solutions**

Turn octobox into a "progressive web app" ([some ios notes](https://dockyard.com/blog/2017/09/27/encouraging-pwa-installation-on-ios)), 
- This would increase screen space available to octobox when used after being added to the homescreen.
- It would also mean octobox is treated as a separate application making it easier to switch between apps.

**Comment by martinklepsch** [comment]
We ([Status Open Bounty](https://openbounty.status.im/)) would like to put a bounty on this issue.

To make this happen two questions come up:
- Are the maintainers of octobox fine with this?
- What do you think about the amount of work? 4hrs? 1-2 days?

**Comment by andrew** [comment]
@martinklepsch yeah happy to have a bounty on it, I'm not particularly familiar with progressive web app manifests but it doesn't look like it'd take more than a day

**Comment by andrew** [comment]
I tried to tackling this on the weekend but it appears that iOS "add to homescreen" web apps that use Oauth for login always bump you back out to mobile safari: https://discussions.apple.com/message/33306161#33306161

Behaviour doesn't seem to have changed in iOS 12 either, so not sure if this is something that can be fixed yet.

**Comment by andrew** [comment]
A possible work-around posted on twitter, although I suspect it won't work for GitHub Oauth as you can't put github.com in an iframe: https://mobile.twitter.com/j_mcnally/status/1051720250402717697

**Comment by andrew** [comment]
@freeformz I don't believe there's a fix for it on iOS at the moment

**Comment by felixbuenemann** [comment]
Looks like it will work with iOS 12.2:

https://twitter.com/mhartington/status/1089292031548145666

**Comment by BenJam** [comment]
👍can't wait! might have to sneak in a few mobile specific behaviours (hello swipe actions)
